pkg/routeagent/controllers/route/route_test.go

```diff
@@ -0,0 +1,46 @@
+package routecontroller
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("RouteController", func() {
+
+	Describe("Function populateCidrBlockList", func() {
+
+		Context("subnets does not contain CIDR blocks", func() {
+			It("should append CIDR block to subnets", func() {
```

Comment:
Describe, Context and It should read as natural language together, how about:
 
    Context("When input CIDR blocks are not present in the existing subnets", func() {
        It("should append the CIDR blocks to subnets", func() {